chsdi/models/vector/stopo.py

```diff
@@ -2567,12 +2623,22 @@ class ga25_polygon_main(Base, Vector):
     description = Column('description', Text)
     tecto = Column('tecto', Text)
     url_legend = Column('url_legende', Text)
+    __maxscale__ = 70000
     the_geom = Column(Geometry(geometry_type='GEOMETRY',
                                dimension=2, srid=21781))
 
 register('ch.swisstopo.geologie-geologischer_atlas', ga25_polygon_main)
```

Comment:
Could you factorize the above and put the `__queryable_attributes__ = []` on the base class?